schema/soo/ofc/custom_properties/CT_Property.go

```diff
@@ -415,18 +414,10 @@ lCT_Property:
 				}
 			case xml.Name{Space: "http://schemas.openxmlformats.org/officeDocument/2006/docPropsVTypes", Local: "filetime"},
 				xml.Name{Space: "http://purl.oclc.org/ooxml/officeDocument/docPropsVTypes", Local: "filetime"}:
-				filetimeStr := new(string)
-				if err := d.DecodeElement(filetimeStr, &el); err != nil {
+				m.Filetime = new(time.Time)
```

Comment:
Is this intentional?  Seems like previously was assuming a specific time format
"2006-01-02T15:04:05Z07:00"
and falling back to
"02-01-2006T15:04:05Z07:00"

now is assuming d.DecodeElement takes care of this?

Reply:
MS Office throws this attribute out when it is string and leaves as it is when it is date. DecodeElement is not deleted, it is just isn't shown here by some reason.